.travis.yml

```diff
@@ -1,28 +1,33 @@
 language: node_js
+os: linux
 node_js:
   - "8.9.0"
-sudo: required
 services:
   - docker
-cache:
```

Comment:
Before there we conflicting `cache` entries, maybe that was a reason things were not caching correctly.